boards/arm/adafruit_feather_stm32f405/adafruit_feather_stm32f405_defconfig

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ARM=y
+CONFIG_BOARD_ADAFRUIT_FEATHER_STM32F405=y
+CONFIG_SOC_SERIES_STM32F4X=y
+CONFIG_SOC_STM32F405XG=y
+# 168MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=168000000
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+CONFIG_SERIAL=y
+
+# console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# enable pinmux
+CONFIG_PINMUX=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# clock configuration
+CONFIG_CLOCK_CONTROL=y
+
+# Clock configuration for Cube Clock control driver
+CONFIG_CLOCK_STM32_HSE_CLOCK=12000000
+CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL=y
+# use HSE as PLL input
+CONFIG_CLOCK_STM32_PLL_SRC_HSE=y
+# produce 168MHz clock at PLL output
+CONFIG_CLOCK_STM32_PLL_M_DIVISOR=12
+CONFIG_CLOCK_STM32_PLL_N_MULTIPLIER=336
+CONFIG_CLOCK_STM32_PLL_P_DIVISOR=2
+CONFIG_CLOCK_STM32_PLL_Q_DIVISOR=7
+CONFIG_CLOCK_STM32_AHB_PRESCALER=1
+CONFIG_CLOCK_STM32_APB1_PRESCALER=4
+CONFIG_CLOCK_STM32_APB2_PRESCALER=2
+
+#enable pwm
+CONFIG_PWM=y
```

Comment:
Aside from minimum required for basic operations (GPIO, clock, UART), no peripheral should be enabled in default board configuration.